### DISCUSSION TITLE
PaddleOCR/en/FAQ

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#13809] [Q&A]
# PaddleOCR/en/FAQ

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/en/FAQ.html

**Comment by abhijeetsourav** [comment]
how to convert and extract only parseq ( Text Recognition Model ) to ONNX
